doc/usersguide/codes/openmc.rst

```diff
@@ -41,6 +41,38 @@ then add them to a group called, "mat:Vacuum":
 
     CUBIT> group "mat:Vacuum" add vol 4 to 18
 
+
+Assigning Materials by Name (recommended)
+-----------------------------------------
+
+OpenMC materials can also be assigned by name:
+::
+
+    CUBIT> group "mat:fuel" add vol 4 to 18
+
+For this example, a material with the name attribute "fuel" must be present in
+OpenMC's ``materials.xml`` file:
+::
+   <materials>
+     <material id="40" name="fuel">
+       <density units="g/cc" value="11" />
+       <nuclide ao="1.0" name="U235" />
+     </material>
+   </materials>
+
+This method for assigning materials is recommended for use with OpenMC as it
+provides a more verbose description of the material definition and purpose.
+
+This method of assignment also allows for straighforward model conversion to the
```

Comment:
What happens if the same material is defined in the `materials.xml` file and the embedded MaterialLibrary?  Or the same material name, but different definition?

Reply:
I'll clarify that situation.